### DISCUSSION TITLE
what is this

### DISCUSSION THREAD

**Discussion by chaostheunderscore** [Leather128/LeathersFunkinEngine#61] [Q&A] [answered]
i dont know what to do for the "offset" txt file, like what do i put inside the txt file?
![image](https://user-images.githubusercontent.com/89908006/143213394-627ce008-ea75-4fa9-99d3-91c68767a7ce.png)

**Comment by SrtHero278** [comment, marked as answer]
it just makes offsets of the characters. You type in the animation name, the x offset, and then the y offset.
Here is what bf's offset file looks like:
![image](https://user-images.githubusercontent.com/93018286/143381236-942ebb68-9fa4-429c-949f-1ffd9758d359.png)

**Reply by chaostheunderscore** [reply]
thanks ill try it
